GDPR/GDPRManager.swift

```diff
@@ -67,50 +62,69 @@ public class GDPRManager {
                                                       showTermsSwitch: true)
         return ConfirmationView(viewModel: confirmationViewModel!, onConfirm: onConfirm)
     }
-    
+
     // MARK: - Helper Functions
-    
+
     public func setURLs(termsURL: URL, privacyPolicyURL: URL) {
         currentStatus = PersistenceManager.shared.retrieveStatus()
         self.termsURL = termsURL
         self.privacyPolicyURL = privacyPolicyURL
     }
-    
+
     public func setServices(services: [Service]) {
         guard let currentStatus = currentStatus else { return }
+        assert(!services.map { $0.id }.hasDuplicates, "The ids of the services have to be unique" )
         for service in services {
             for singleService in currentStatus.services where service.id == singleService.id {
                 singleService.description = service.description
                 singleService.name = service.name
                 singleService.supportDeletion = service.supportDeletion
                 return
             }
-            self.currentStatus?.services.append(ServiceModel(id: service.id, name: service.name, description: service.description, supportDeletion: service.supportDeletion, isOptIn: false))
```

Comment:
shouldn't isOptIn == false let the toggle be "on" by default?

Reply:
I don't understand the question